src/OSX/Avalonia.MonoMac/MonoMacPlatform.cs

```diff
@@ -39,13 +43,43 @@ public static void Initialize()
 
         }
 
+
+        /// <summary>
+        /// See "Using POSIX Threads in a Cocoa Application" section here:
+        /// https://developer.apple.com/library/content/documentation/Cocoa/Conceptual/Multithreading/CreatingThreads/CreatingThreads.html#//apple_ref/doc/uid/20000738-125024
+        /// </summary>
+        class ThreadHelper : NSObject
+        {
+            private readonly AutoResetEvent _event = new AutoResetEvent(false);
+            [Export("doNothing")]
```

Comment:
Do we want to do a private constant here like we do below for the selector name?